standard/1.3/workflow-definition.schema.json

```diff
@@ -0,0 +1,262 @@
+                "label",
+                "icon",
+                "parameters"
+            ],
+            "additionalProperties": false,
+            "title": "Step definition"
+        },
+        "tag": {
+            "type": "string",
+            "examples": [
+                "speech_to_text"
+            ],
+            "pattern": "^(.*)$",
+            "title": "The Items Schema"
+        }
+    },
+    "properties": {
+        "identifier": {
+            "examples": [
+                "speech_to_text"
+            ],
+            "pattern": "^([a-z_]+)$",
+            "title": "The Identifier of the workflow, used to reference it",
+            "type": "string"
+        },
+        "reference": {
+            "title": "External reference identifier for this workflow, can be UUID, string etc.",
+            "type": "string"
+        },
+        "label": {
+            "$id": "#/properties/label",
+            "examples": [
+                "speech_to_text"
+            ],
+            "pattern": "^(.*)$",
+            "title": "The label of the workflow, used as displayed name",
+            "type": "string"
+        },
+        "icon": {
+            "examples": [
+                "subtitles"
+            ],
+            "pattern": "^([a-z_]+)$",
+            "title": "The icon used with the label. One of https://material.io/resources/icons/",
+            "type": "string"
+        },
+        "start_parameters": {
+            "items": { "$ref": "#/definitions/start_parameter" },
+            "title": "Definition of available parameters to start the workflow definition",
+            "type": "array"
+        },
+        "parameters": {
+            "items": { "$ref": "parameter.schema.json#" },
+            "title": "Storage of dynamic parameters during process of the workflow instance",
+            "type": "array"
+        },
+        "steps": {
+            "items": { "$ref": "#/definitions/step" },
+            "title": "Steps to represent the complete process",
+            "type": "array"
+        },
+        "tags": {
+            "items": {"$ref": "#/definitions/tag" },
+            "title": "List of tags to classify the worklow",
+            "type": "array"
+        },
+        "version_major": {
+            "title": "Major version of this Workflow definition",
+            "type": "integer"
+        },
+        "version_minor": {
+            "title": "Minor version of this Workflow definition",
+            "type": "integer"
+        },
+        "version_micro": {
+            "title": "Micro version of this Workflow definition",
+            "type": "integer"
+        }
+    },
+    "required": [
+        "identifier",
+        "label",
+        "icon",
+        "version_major",
+        "version_minor",
+        "version_micro",
+        "tags",
+        "parameters",
+        "steps"
+    ],
+    "additionalProperties": false,
+    "title": "Schema of Workflow Definition for Media Cloud AI",
+    "type": "object"
```

Comment:
change reference on documentation:
https://github.com/media-cloud-ai/media-cloud-ai.github.com/blob/master/parameter.schema.md
and here:
https://github.com/media-cloud-ai/media-cloud-ai.github.com/blob/master/workflow-definition.schema.md

Reply:
Updated